design/media_types.go

```diff
@@ -68,6 +68,7 @@ var position = a.Type("workItemReorderPosition", func() {
 	})
 	a.Attribute("direction", d.String, "Direction of the place of the reorder workitem. Above should be used to place the reorder workitem(s) above workitem with id equal to position.id. Below should be used to place the reorder workitem(s) below workitem with id equal to position.id", func() {
 		a.Enum("above", "below", "top", "bottom")
+		a.Description("Above places the reorder item(s) above the reference workitem. Below places the reorder item(s) below the reference workitem. Top places the reorder workitem(s) at the Topmost position of the list. Bottom places the reorder item(s) at the bottom of the list.")
```

Comment:
I meant describe Top/Bottom in the Description that was part of the Attribute already.